advanced/helm/integrator-with-analytics/integrator-with-analytics/values.yaml

```diff
@@ -1,10 +1,94 @@
-username: ""
-password: ""
-email: ""
-namespace: "wso2"
-svcaccount: "wso2svc-account"
-serverIp: ""
-sharedDeploymentLocationPath: ""
-sharedTenantsLocationPath: ""
-
-# cloudProvider: Azure
+# Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+wso2:
+  mysql:
+    enabled: true
+    host: wso2ei-rdbms-service-mysql
+    username: wso2carbon
+    password: wso2carbon
+    driverClass: com.mysql.jdbc.Driver
+    validationQuery: "SELECT 1"
+  subscription:
+    username: ""
+    password: ""
+  deployment:
+    wso2ei:
+      imageName: "wso2ei-integrator"
+      imageTag: "6.5.0"
+      replicas: 1
+      minReadySeconds: 30
```

Comment:
@Savidude, @ThilinaManamgoda as per [1] we may have to either increase the default value set for the `minReadySeconds` to a value greater than what is set for the Readiness Probe grace period. WDYT?

[1]: [MinReadySeconds vs Readiness Probes](https://stackoverflow.com/questions/53239081/how-does-minreadyseconds-affect-readiness-probe)

Reply:
Fixed https://github.com/wso2/kubernetes-ei/pull/180/commits/4eab0894505a022e409f632e44196a919575ceb1